requirements.txt

```diff
@@ -1,10 +1,12 @@
 beautifulsoup4
 dill
+flake8==3.5.0
```

Comment:
Why pin to `3.5.0`?

Reply:
Oops, sorry; force of habit from the projects I work on.  We use pyup.io to auto-manage/update dependencies, and so we pin nearly everything.

I'll unpin.